### ISSUE TITLE
kubernetes_role_binding resource not recognized

### ISSUE THREAD

**Issue by snstanton** [VladRassokhin/intellij-hcl#190]
### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [x] IDE version (`Help->About->Copy to Clipboard`)

PyCharm 2018.3.3 (Professional Edition)
Build #PY-183.5153.39, built on January 9, 2019
JRE: 1.8.0_152-release-1136-b29 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.2

 - [x] intellij-hcl plugin version (`Settings->Plugins`)
v0.6.13
 - [x] Terraform version (`terraform -v`)
0.11.11
kubernetes provider version 1.5.0

### Terraform Configuration Files
```hcl
provider "kubernetes" {
  version = "~> 1.5"
}
resource "kubernetes_role_binding" "myrole" {
  metadata {
    name = "myrole"
    namespace = "mynamespace"
  }
  subject {
    api_group = ""
    kind = "ServiceAccount"
    name = "myaccount"
    namespace = "mynamespace"
  }
  role_ref = {
    api_group = "rbac.authorization.k8s.io"
    kind = "ClusterRole"
    name = "edit"
  }
}
```

### Expected Behavior

kubernetes_role_binding should be recognized as a standard resource of the kubernetes provider

### Actual Behavior
The resource is not recognized, so `metadata` and `subject` properties are flagged as unknown:

![image](https://user-images.githubusercontent.com/209370/51939863-6154fd00-23c5-11e9-96fb-fed248c24478.png)

**Comment by VladRassokhin** [comment]
`kubernetes_network_policy`, `kubernetes_role`, `kubernetes_role_binding` resources and `kubernetes_secret` datasource were added and released as art of kubernetes provider after latest plugin release, so it does not contain information about them. Will be fixed automatically with next update

**Comment by VladRassokhin** [comment]
Fixed in 0.6.14
